src/presentation/index.ts

```diff
@@ -49,7 +50,9 @@ function startSofieServer(): void {
 
 function attachExpressServerToPort(port: number): void {
   new SofieServer().server.listen(port, () => {
-    return console.log(`### Express is listening at http://localhost:${port}`)
+    const loggerService: LoggerService = new LoggerService()
+    loggerService.tag('index')
```

Comment:
This is the presentation layer index file, not just index.

Reply:
Have updated the text for the tag.